game.py

```diff
@@ -20,7 +20,38 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    
+    snowman_word_dic = build_letter_status_dict(snowman_word)
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_count = 0
+    wrong_guesses_list = set()
+
+    while wrong_guesses_count < SNOWMAN_MAX_WRONG_GUESSES:
```

Comment:
This is one place where that `len()` function can really come in handy! I would try something along the lines of: 
```suggestion
    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
```